maquetado/scss/book-list.scss

```diff
@@ -0,0 +1,25 @@
+.book-list {
+  display: flex;
+  flex-wrap: wrap;
+  width: 60%;
+}
+
+.book-thumbnail {
+  box-shadow: 1px 1px 2px;
+  padding: 10px;
+  margin: 30px;
+  height: 281px;
+  width: 199px; 
+}
+
+.book-thumbnail-image {
+  display: block;
+  margin-left: auto;
+  margin-right: auto;
```

Comment:
You can achieve the same by doing `margin: 0 auto;` but I would suggest you to use flex instead to center your image.